fiftyone/factory/__init__.py

```diff
@@ -8,26 +8,28 @@
 from enum import Enum
 
 
-class DelegatedOpPagingParams(object):
-    class SortByField(Enum):
-        QUEUED_AT = "queued_at"
-        COMPLETED_AT = "completed_at"
-        STARTED_AT = "started_at"
-        FAILED_At = "failed_at"
-        OPERATOR_NAME = "operator"
+class SortByField(Enum):
+    QUEUED_AT = "queued_at"
+    COMPLETED_AT = "completed_at"
+    STARTED_AT = "started_at"
+    FAILED_At = "failed_at"
+    OPERATOR_NAME = "operator"
+
+
+class SortDirection(Enum):
+    ASCENDING = 1
+    DESCENDING = -1
 
-    class SortDirection(Enum):
-        ASCENDING = 1
-        DESCENDING = -1
 
+class DelegatedOpPagingParams(object):
     def __init__(
         self,
-        skip: int = 0,
-        limit: int = 10,
         sort_by: SortByField = SortByField.QUEUED_AT,
         sort_direction: SortDirection = SortDirection.DESCENDING,
+        skip: int = 0,
+        limit: int = 10,
```

Comment:
@nebulae what about making these be `None` by default? The fact that App usages of these params will likely want paging doesn't mean that programmatic usages will necessarily want paging